src/org/labkey/test/util/ListHelper.java

```diff
@@ -507,14 +507,20 @@ public RangeValidator(String name, String description, String message, RangeType
     @Deprecated
     public static class ListColumn extends FieldDefinition
     {
-        public ListColumn(String name, String label, ListColumnType type, String description, String format, LookupInfo lookup, FieldValidator validator, String url, Integer scale)
+        public ListColumn(String name, String label, ListColumnType type, String description, String format, LookupInfo lookup, FieldValidator<?> validator, String url, Integer scale)
```

Comment:
I realize we may not have the time right now, but since this is deprecated, shouldn't we stop fixing it and move existing code to it's replacement?

Reply:
We definitely should. I started pulling that thread but quickly realized it would take over the PR.